airflow/ui/src/pages/DagsList/Dag/DagDocumentation.tsx

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Heading } from "@chakra-ui/react";
+import Markdown from "react-markdown";
+
+import type { DAGDetailsResponse } from "openapi/requests/types.gen";
+import { Dialog } from "src/components/ui";
+
+type DagDocumentationModalProps = {
+  docMd: DAGDetailsResponse["doc_md"];
+  onClose: () => void;
+  open: boolean;
+};
+
+export const DagDocumentation: React.FC<DagDocumentationModalProps> = ({
+  docMd,
+  onClose,
+  open,
+}) => (
+  <Dialog.Root onOpenChange={onClose} open={open} size="full">
```

Comment:
```suggestion
  <Dialog.Root onOpenChange={onClose} open={open} size="md">
```

`full` is way too big

Reply:
Thanks. Done and updated screenshot in description. Personally `lg` looks wider and better.